src/main/java/com/github/manolo8/darkbot/modules/DummyExceptionModule.java

```diff
@@ -0,0 +1,20 @@
+package com.github.manolo8.darkbot.modules;
+
+public class DummyExceptionModule extends DummyModule {
+
+    private final String feature;
+
+    public DummyExceptionModule(String feature) {
+        this.feature = feature;
+    }
+
+    @Override
+    public String getStatus() {
+        return "Module: " + feature + ", occurred with critical error!";
```

Comment:
```suggestion
        return "Module '" + feature + "' failed with critical error!\nSee plugins tab for more info";
```
I'd add a 2nd line to remind them to just go look at that